### PR TITLE
chore: 🤖 add preferred endpoitns

### PR DESCRIPTION
**Pull request by DhariniJeeva**

### CODE REVIEW

File: addons/core/translations/resources/en-us.yaml [hunk 1]
```diff
@@ -111,6 +111,9 @@ host-set:
     filter: 
       label: Filter
       help: Create a filter to select resources using values such as tag name or tag value. Filter format is specific to each provider
+    preferred_endpoints:
```
Comment (randallmorey) [inline addons/core/translations/resources/en-us.yaml:114]: Can we add a `placholder` as well and make it a realistic example of an cidr or dns address?  Then add it to the field.

**Review by randallmorey — CHANGES_REQUESTED** [review]

File: addons/core/translations/resources/en-us.yaml (hunk 1 above)
Comment (DhariniJeeva) [reply addons/core/translations/resources/en-us.yaml:114]: sure. would something like `43.33.85.22` work?

Comment (DhariniJeeva) [reply addons/core/translations/resources/en-us.yaml:114]: added!

File: ui/admin/app/components/form/host-set/aws/index.hbs [hunk 2]
```diff
@@ -74,6 +74,94 @@
...
+        <table.body as |body|>
+          {{#each @model.preferred_endpoints as |endpoint i|}}
+            <body.row as |row|>
+              <row.cell>
+                <form.input
+                  @type='text'
+                  @value={{endpoint.value}}
+                  @error={{@model.errors.preferred_endpoints}}
+                  @contextual={{true}}
+                  @placeholder='cidr:2001:4860:4860::8888/32'
```
Comment (randallmorey) [inline ui/admin/app/components/form/host-set/aws/index.hbs:106]: Does `placeholder` not work?  I would have thought a regular HTML attribute would suffice.

**Review by randallmorey — CHANGES_REQUESTED** [review]

File: ui/admin/app/components/form/host-set/aws/index.hbs (hunk 2 above)
Comment (DhariniJeeva) [reply ui/admin/app/components/form/host-set/aws/index.hbs:106]: oops it does work! I tried that initially and had some typo and went with the input attr route. I can update it now

File: ui/admin/app/components/form/host-set/aws/index.hbs [hunk 3]
```diff
@@ -74,6 +74,94 @@
...
+              </row.cell>
+            </body.row>
+          {{/each}}
+
+          <body.row as |row|>
+            <row.cell>
+              <form.input
+                @type='text'
+                @value={{@model.preferred_endpoint}}
+                @placeholder='cidr:2001:4860:4860::8888/32'
```
Comment (calcaide) [inline ui/admin/app/components/form/host-set/aws/index.hbs:135]: Already mentioned on previous comment about palceholder

File: ui/admin/app/components/form/host-set/azure/index.hbs [hunk 4]
```diff
@@ -74,6 +74,94 @@
...
+        <table.body as |body|>
+          {{#each @model.preferred_endpoints as |endpoint i|}}
+            <body.row as |row|>
+              <row.cell>
+                <form.input
+                  @type='text'
+                  @value={{endpoint.value}}
+                  @error={{@model.errors.preferred_endpoints}}
+                  @contextual={{true}}
+                  @placeholder='cidr:2001:4860:4860::8888/32'
```
Comment (calcaide) [inline ui/admin/app/components/form/host-set/azure/index.hbs:106]: Same palceholder as mentioned above

**Comment by DhariniJeeva** [comment]
changes are updated! ty both!

File: addons/rose/addon/components/rose/form/input/index.stories.mdx [hunk 5]
```diff
@@ -13,7 +13,7 @@ Renders a simple form input.
 <Rose::Form::Input
   @type="email"
   @name="email"
-  @value="info@example.com"
```
Comment (randallmorey) [inline addons/rose/addon/components/rose/form/input/index.stories.mdx:16]: Can this be changed back to what it originally was?

**Review by randallmorey — CHANGES_REQUESTED** [review]

File: addons/rose/addon/components/rose/form/input/index.stories.mdx (hunk 5 above)
Comment (DhariniJeeva) [reply addons/rose/addon/components/rose/form/input/index.stories.mdx]: done!

**Review by calcaide — APPROVED** [review]
LGTM

**Review by randallmorey — APPROVED** [review]

**Pull request merged**